Adding ng2 component lab and sample buttons

islavi opened the pull request:
Lets go over it tomorrow.

nomrik reviewed (changes requested):
a couple of things:

- Pull requests are to be done from a feature branch on the fork, after you synchronised your fork's master with the original master, and then rebased it onto your feature branch. Pull requests that are from a fork's master branch are not accepted, as they potentially can change history. Please see contribution guide for more details. After the fixes to the PR are done, it should be done from a new feature branch, and you will need to reset your fork's master to the state of the original one.

- Why is webpack now installed as a dependency? I don't see it's used somewhere in our build process. Storybook uses it internally, I believe the angular components lab does the same and has webpack in it's dependencies. Webpack is a pricey dependency performance-wise, so if we don't use it, it should be removed from package.json

- Why did you change our webpack config inside storybook?

- Are you sure forking components-lab is a good idea? You will diverge from it/have to sync it manually every time it updates. It's better to propose the changes you need as a PR to the original repository.

Thanks

islavi commented:
@nomrik @zivl 

1. Currently did not created branch for this, next features will be in separate branch. We can go over the changes together, they are mainly in different folders and do not touch same code.
2. Webpack is installed as devDependencies and not dependencies. It is used by component lab to compile the component written in typescript in our code. Component lab merge our webpack configuration with component lab configuration.
3. I added suffix "-loader" to the loaders. Because automatic -loader module name extension removed. See: [https://webpack.js.org/guides/migrating/#automatic-loader-module-name-extension-removed](https://webpack.js.org/guides/migrating/#automatic-loader-module-name-extension-removed)
4. Component lab does not update very often, people not updating it. They have some bugs that I fixed and thought to propose, but the real issue is that they are using angular 4 which is not good for us (we are using angular 2). This is why I did not propose my changes, and created fork.

Thanks for reviewing :)
